### DISCUSSION TITLE
Unable to reset email or password due to Firebase site not found problem.

### DISCUSSION THREAD

**Discussion by MatthewCoetzee412** [firebase/flutterfire#3595] [Q&A]
Hi Everyone :wave: ,

I recently attempted to implement the ability to reset a user's email and password in my Flutter application. It sent the email successfully but shortly after I clicked on the URL link I was met with a "Site Not Found". Multiple options on why this message was being displayed were listed below but they had to do with Firebase Hosting or using a custom domain which I use neither of them. I viewed multiple Youtube tutorials to see if they ran into similar problems but the link took them to a elegantly formatted website where they were able to reset their email or password. I am not sure why this problem is occurring but it might be due to the fact that I need a custom domain, although none of the Youtubers required it.

Below is the error I'm faced with on the site:
![image](https://user-images.githubusercontent.com/42652264/93689643-c72d5580-fad0-11ea-8db2-ddd4834e0d68.png)

Below is the code:

Firebase Auth:
```yaml
  firebase_auth: ^0.17.0-dev.2
```

Reset password:
```Dart
  Future<void> resetPassword(String email) async {
    await _authInstance.sendPasswordResetEmail(email: email);
  }
```

Reset Email:
```Dart
  Future<void> resetEmailAddress(
      String newEmail, String oldEmail, String password) async {
    var authResult = await _authInstance.signInWithEmailAndPassword(
        email: oldEmail, password: password);
    await authResult.user.updateEmail(newEmail);
    await _instance
        .collection('users')
        .doc(authResult.user.uid)
        .update({'email': newEmail});
  }
```

Lastly I'm sorry for the lack of error checking, and I would greatly appreciate it if anyone would be kind enough to update my code! 

Thank you for you time!
- Matt

**Comment by artimmy** [comment]
did you get the solution???

**Reply by antojosu** [reply]
Did you found any workaround?
